File: php/delivery-api/GetTaxonomyGroup.php
<?php
// DocSection: delivery_api_get_taxonomy_group
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('<YOUR_PROJECT_ID>');

$taxonomyGroup = $client->getTaxonomy('personas');
// EndDocSection
?>

File: java/delivery-api/GetTaxonomyGroup.java
// DocSection: delivery_api_get_taxonomy_group
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("<YOUR_ENVIRONMENT_ID>");

// Gets a specific taxonomy group
CompletionStage<TaxonomyGroup> personas = client.getTaxonomyGroup("personas");
// EndDocSection